openc3-init/plugins/packages/openc3-tool-tlmviewer/src/tools/TlmViewer/Openc3Screen.vue

```diff
@@ -404,13 +324,37 @@ export default {
                 this.globalSubsettings.push(parameters)
                 break
               default:
-                this.processWidget(keyword, parameters)
+                this.processWidget(keyword, parameters, line, lineNumber)
                 break
             } // switch keyword
           } // if keyword
         }
       )
-      this.applyGlobalSettings(this.layoutStack[0].widgets)
+      // This can happen if there is a typo in a layout widget with a corresponding END
+      if (typeof this.layoutStack[0] === 'undefined') {
```

Comment:
We're still not handling undefined keywords well but this prevents weird stuff when a layout widget is mis-spelled